scalableminds/webknossos
# Toggle trees

### Mailable description of changes
- The visibility of each tree in a tracing can now be toggled in the tree tab. Toggling trees via "1" and "2" still works and also works in the arbitrary modes now.

### Steps to test:
- ensure the toggling works via the checkboxes as well as the new buttons in the tree-view
- ensure that only toggling tree visibility and saving doesn't send anything to the server
- ensure that node picking (shift click) still works and won't select invisible nodes

### Issues:
- fixes #1921 
- fixes #1411

------

I also removed the warning when toggling visibility via the shortcuts, since I think that checkboxes make it more than clear when all trees are hidden.
